packages/primitives/src/Menu/MenuItem.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import { Box } from '../Box'
+
+type MenuItemProps = {
+  children: React.ReactNode
+} & React.ComponentProps<typeof Box>
+
+export const MenuItem = React.forwardRef<HTMLLIElement, MenuItemProps>(
+  function MenuItem(
+    {
+      children,
+      variantKey = 'menuitem',
+      variant,
```

Comment:
```suggestion
      variantKey = 'menu',
      variant = 'menuItem',
```

@alexanderchan what do you think about nesting this in the `menu` variantKey` similar to how we do with Dialogs?